.github/CODEOWNERS

```diff
@@ -596,6 +596,11 @@ app/uploaders/vets_shrine.rb @department-of-veterans-affairs/va-api-engineers @d
 app/validators/token_util.rb @department-of-veterans-affairs/lighthouse-banana-peels
 app/sidekiq/account_login_statistics_job.rb @department-of-veterans-affairs/vsp-identity
 app/sidekiq/bgs @department-of-veterans-affairs/benefits-dependents-management @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
+app/sidekiq/bgs/job.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```

Comment:
Please remove this line

Reply:
Lines 599-603 can be removed.

Reply:
Should I not do the same for the spec/bgs folder?

Reply:
Actually you have me confused with that statement, because if so then really I should remove most lines from the PR